tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js

```diff
@@ -0,0 +1,9 @@
+export const sumAll = ( arr ) => {
+  const minimum = Math.min( ...arr );
+  const maximum = Math.max( ...arr );
+  let sum = 0;
+  for ( let i = minimum; i <= maximum; i++) {
+    sum += i;
+  }
+  return sum;
```

Comment:
just FYI - you can solve this without Math functions in one loop. Just think if you really need to loop from minimum to maximum.